allennlp/semparse/worlds/wikitables_variable_free_world.py

```diff
@@ -147,34 +147,52 @@ def get_agenda(self):
         agenda_items = []
         question_tokens = [token.text for token in self.table_context.question_tokens]
         question = " ".join(question_tokens)
+
+        if "at least" in question:
```

Comment:
Can you add 4-5 test cases in the `test_get_agenda` method of `allennlp/tests/semparse/worlds/wikitables_variable_free_world_test.py` that esure the correct rules are being triggered for "at least", "at most", "no greater than", "top" and "bottom"?

Reply:
sure, will do it now